scripts/update-maintenance-issue.ts

```diff
@@ -0,0 +1,61 @@
+import { Octokit } from 'octokit';
+
+const githubToken = process.env['GITHUB_TOKEN'];
+const pullRequestNumber = parseInt(process.env['PR_NUMBER']!, 10);
+
+const repoConfig = {
+  owner: 'sbb-design-systems',
+  repo: 'sbb-angular',
+};
+
+const issuePath = {
+  ...repoConfig,
+  issue_number: 1346,
+};
+
+const prPath = {
+  ...repoConfig,
+  pull_number: pullRequestNumber,
+};
+
+export class MaintenanceIssueUpdater {
+  constructor(private _octokit: Octokit, private _now: Date) {}
+
+  async run() {
+    const issue = await this._octokit.rest.issues.get(issuePath);
+    const pr = await this._octokit.rest.pulls.get(prPath);
+
+    if (!issue.data.body) {
+      throw new Error('Could not load issue body');
+    }
+
+    if (!pr.data.title || !pr.data.html_url) {
+      throw new Error('Could not load pull request');
+    }
+
+    const hint = '**Unable to merge the following pull requests into the maintenance branch**';
+    const dateInfo = `${this._now.toISOString()}`;
+    let openTasks = this._extractOpenTasks(issue.data.body);
+    openTasks.push(`- [ ] [${pr.data.title}](${pr.data.html_url})`);
+    openTasks = [...new Set(openTasks)];
```

Comment:
maybe extract method as it is in extract, but this time the opposite (include duplication removal and line joining?